File: CHALLENGES/1. deDup.java
import java.util.Arrays;

class Main {
  public static int[] deDup(int[] a) {
    return Arrays.stream(a).distinct().toArray();
  }
  
  public static void main(String[] args) {
    int [] a = new int []{1, 2, 3, 3, 3, 3, 3};
    int [] b = deDup(a);
    for(int i: b){
    System.out.print(i + " ");
    }
    System.out.println();
  }
}

File: CHALLENGES/1. deDup.cs
using System;
using System.Linq;

class MainClass {
  public static int[] deDup(int[] a) {
    return a.Distinct().ToArray();
  }
  
  public static void Main (string[] args) {
    int [] a = new int []{1, 2, 3, 3, 3, 3, 3};
    var result = deDup(a);
    foreach(int i in result){
      Console.Write(i + " ");
    }
    Console.WriteLine();
  }
}
